keyboards/nightly_boards/octopad/readme.md

```diff
@@ -0,0 +1,18 @@
+# Nightly Boards Octopad
+
+![octopad](https://i.imgur.com/MkSzzGBl.jpg)
+
+An 8 key macropad with 2 rotary encoders
+
+* Keyboard Maintainer: [Neil Brian Ramirez](https://github.com/NightlyBoards)
+* Hardware Supported: atmega32u4
+
+Make example for this keyboard (after setting up your build environment):
+
+	make nightly_boards/octopad:default
+	
+Flashing example for this keyboard:
+
+	make nightly_boards/octopad:default:flash
+
```

Comment:
Can you add "clear instructions on how to reset the board into bootloader mode" as per the [PR checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?